plugins/pagerduty/komand_pagerduty/actions/create_user/action.py

```diff
@@ -16,20 +16,28 @@ def __init__(self):
         )
 
     def run(self, params={}):
-        """Run action"""
-        self.logger.info("Creating user %s", params)
-        user = pypd.User.create(data=params, from_email=params["from_email"])
-
-        if user:
-            user = normalize_user(user.json)
-        else:
-            user = None
-
-        self.logger.debug("Returned: %s", user)
-
-        return {"success": not not user, "user": user or empty_user}
-
-    def test(self):
-        """Test action"""
+        """Trigger event"""
```

Comment:
This is just a pedantic request really but we can remove these type of comments

Reply:
these comments have now been removed across all actions